web/js/waste.js

```diff
@@ -86,6 +88,7 @@ $(function() {
       $('#pro_rata_cost').text(pro_rata_cost.toFixed(2));
     }
     $('#modify #bins_wanted, #modify #current_bins').on('change', modify_cost);
+    $(document).ready(modify_cost, bin_cost_new);
```

Comment:
I don't think you can supply multiple functions to read(). Also bit worried that the modify code doesn't expect to be run on the new/renew page, and vice-versa? Also on a new page, don't you need to call it with bin_cost_new(1)?